core/src/main/java/org/elasticsearch/cluster/routing/OperationRouting.java

```diff
@@ -55,19 +54,16 @@ public OperationRouting(Settings settings, AwarenessAllocationDecider awarenessA
     }
 
     public ShardIterator indexShards(ClusterState clusterState, String index, String type, String id, @Nullable String routing) {
```

Comment:
@areek can we remove all the changes to this class now? or is there some stuff here we need?